File: lib/src/main/java/de/tum/in/tumcampusapp/models/TokenConfirmation.java
package de.tum.in.tumcampusapp.models;

import org.simpleframework.xml.Root;
import org.simpleframework.xml.Text;

@Root(name = "confirmed")
public class TokenConfirmation {
    @Text
    private String confirmed = "";

    public boolean isConfirmed() {
        return confirmed.equals("true");
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/api/tumonline/model/TokenConfirmation.kt
package de.tum.`in`.tumcampusapp.api.tumonline.model

import com.tickaroo.tikxml.annotation.TextContent
import com.tickaroo.tikxml.annotation.Xml

@Xml(name = "confirmed")
data class TokenConfirmation(@TextContent val confirmed: String) {
    val isConfirmed: Boolean
        get() = "true" == confirmed
}
